vaadin-testbench-shared/src/main/java/com/vaadin/testbench/ElementQuery.java

```diff
@@ -258,41 +438,44 @@ public ElementQuery<T> withAttribute(String attribute, String value) {
      * <p>
      * For matching the full attribute value, see
      * {@link #withAttribute(String, String)}.
+     * For matching a substring of the attribute value, see
+     * {@link #withAttributeContaining(String, String)}.
      *
      * @param attribute
      *            the attribute name
-     * @param value
+     * @param token
      *            the token to look for
      * @return this element query instance for chaining
      *
      * @see #withAttribute(String, String)
+     * @see #withAttributeContaining(String, String)
      */
-    public ElementQuery<T> withAttributeContaining(String attribute, String value) {
-        attributes.add(new AttributeMatch(attribute, "~=", value));
-        return this;
+    public ElementQuery<T> withAttributeContainingToken(String attribute, String token) {
```

Comment:
The example in the Javadoc should probably mention `withAttributeContainingToken` instead of `withAttributeContaining`

Reply:
Good catch!